conda/dev.yml

```diff
@@ -0,0 +1,9 @@
+name: zppy_dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
@golaz I'm not sure what specific dependencies `zppy` has. I know it doesn't have many since the generated bash scripts do most of the actual work.